core-scala/src/main/scala/com/baeldung/scala/LazyVal.scala

```diff
@@ -0,0 +1,62 @@
+package com.baeldung.scala
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent._
+import scala.concurrent.duration._
+
+class Member {
```

Comment:
Would it be better to rename the class to LazyVal?

Reply:
It's a good suggestion. I have renamed it to LazyVal